StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheetDataSource.swift

```diff
@@ -42,7 +42,7 @@ class CustomerSheetDataSource {
                 await loadFormSpecs()
                 let customerId = try await customerSessionClientSecret.customerId
                 let elementSession = try await elementsSessionResult
-                let paymentOption = customerSessionAdapter.fetchSelectedPaymentOption(for: customerId, customer: elementSession.customer)
```

Comment:
Hm same, this entire file has no tests :(

Reply:
Not having tests for this class was intentional for most functions in this class as they are just switching on legacy ephemeral keys vs customersession. For `loadPaymentMethodInfo` it's debatable whether we want to add tests it seems like we'd end up with a lot of mocked responses so it's unclear how helpful it would be.